templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -0,0 +1,591 @@
+              <li class="p-list__item is-ticked u-hide"></li>
+              <li class="p-list__item is-ticked">Where you are located</li>
+              <li class="p-list__item is-ticked">Where the product has been developed/produced</li>
+            </ul>
+            <p>The exceptions are:</p>
+            <ul class="p-list--divided">
+              <li class="p-list__item is-ticked u-hide"></li>
+              <li class="p-list__item is-ticked">Solutions used for internal purposes are not considered PDEs</li>
+              <li class="p-list__item is-ticked">Pure SaaS solutions will be excluded</li>
+              <li class="p-list__item is-ticked">PDEs that are already regulated by sector-specific regulations will be excluded</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+    </div>
+
+    <div class="u-fixed-width">
+      <hr class="u-hide--small" />
+
+      <div class="u-fixed-width">
+        <div class="p-section--shallow">
+          <h2>The Cyber Resilience Act requirements in a nutshell</h2>
+        </div>
+      </div>
+
+      <div class="p-section--shallow">
+        <div class="row">
+          <div class="col-9 col-start-large-4">
+            <hr class="is-muted" />
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-medium-3 col-3 col-start-large-4">
+            <h3 class="p-heading--5">Vulnerability management</h3>
+          </div>
+          <div class="col-medium-3 col-6">
+            <p>
+              Manufacturers must maintain security throughout the product lifecycle through:
+              <ul class="p-list--divided">
+                <li class="p-list__item is-ticked u-hide"></li>
+                <li class="p-list__item is-ticked">Security patching and maintenance</li>
+                <li class="p-list__item is-ticked">Incident response</li>
+              </ul>
+            </p>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-9 col-start-large-4">
+            <hr class="is-muted" />
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-medium-3 col-3 col-start-large-4">
+            <h3 class="p-heading--5">Risk assessment</h3>
+          </div>
+          <div class="col-medium-3 col-6">
+            <p>
+              Manufacturers must ensure that PDEs:
+              <ul class="p-list--divided">
+                <li class="p-list__item is-ticked u-hide"></li>
+                <li class="p-list__item is-ticked">Have no known exploitable vulnerabilities</li>
+                <li class="p-list__item is-ticked">Are secure by default with minimal attack surface</li>
+                <li class="p-list__item is-ticked">Minimize processing of data</li>
+              </ul>
+            </p>
+          </div>
+        </div>
+
+
+        <div class="row">
+          <div class="col-9 col-start-large-4">
+            <hr class="is-muted" />
```

Comment:
This class is deprecated, we are now using the new class `p-rule--muted`. Can you update all occurrences of this too?
```suggestion
            <hr class="p-rule--muted" />
```